### DISCUSSION TITLE
Server Redirect Log information

### DISCUSSION THREAD

**Discussion by Odie3** [OldUnreal/UnrealTournamentPatches#1554] [Ideas]
Setting up UT File Redirection is fun (not really).  I know after setting it up if I missed a compressed file on my http location I'll see ../system/FileBeingSent.xxx

It would be great if the server logging showed a file being sent via the URL/FileBeingSent.xxx.  Then if the file isn't seen on the http location or there is a timeout, that is logged and the server is falling back on non-redirection default.

It would also be nice if the server logs that Redirection is Enabled or not.  If enabled, then settings are listed in the log after it has determined if Server Redirection is enabled.

Also, I wanted to use http://MyLocation:9090 and that wouldn't work - seems that you can't use a different port in the URL setting?

**Comment by Odie3** [comment]
btw, I do love seeing on the client the number of files and the total received.  It would be nice to see the files sent in the logging and/or console?

**Reply by BearBeardgh** [reply]
maybe create tab in console. xconsole for example. that could work

**Comment by SeriousBuggie** [comment]
In client log window you can see process of obtain files.
Log window work only for windows and wxwidgets launcher for linux.

Server no idea if client download files via redirect. All that goes without involve game server in process.
